requirements/environment.yml

```diff
@@ -1,4 +1,5 @@
 channels:
+  - conda-forge
   - defaults
   - spyder-ide
```

Comment:
Let's remove these two channels from here too. Since all packages required for testing are part of conda-forge, they are not necessary anymore.